pkg/core/native/native_test/neo_test.go

```diff
@@ -318,8 +327,45 @@ func TestNEO_GetAccountState(t *testing.T) {
 			stackitem.Make(amount),
 			stackitem.Make(lub),
 			stackitem.Null{},
+			stackitem.Make(0),
 		}), "getAccountState", acc.ScriptHash())
 	})
+
+	t.Run("lastGasPerVote", func(t *testing.T) {
```

Comment:
It is a nice test, really. But not exactly what I'd expect to see. Let's keep this test and add one more test (a separate one), e.g. let it be `func TestNeo_GetAccountStateInteropAPI(t *testing.T){...}`. Inside this test we should create a tiny smart-contract, deploy it and call, something like this:
```
func TestNeo_GetAccountStateInteropAPI(t *testing.T){
      // some preparation code...
	src := `package callneo
		import (
			"github.com/nspcc-dev/neo-go/pkg/interop/native/neo"
			"github.com/nspcc-dev/neo-go/pkg/interop"
			"github.com/nspcc-dev/neo-go/pkg/interop/util"
		)
		func CallNeo() int {
			accState := neo.GetAccountState(interop.Hash160{` + hashStr + `})
			if accountState == nil {
				panic("nil state")
			}
			return accState.LastGasPerVote
		}`
	ctr := neotest.CompileSource(t, c.Committee.ScriptHash(), strings.NewReader(src), &compiler.Options{
		Name: "callneo_contract",
	})
	e.DeployContract(t, ctr, nil)

	ctrInvoker := e.NewInvoker(ctr.Hash, e.Committee)
	ctrInvoker.Invoke(t, /*the expected LastGasPerVote represented as stackitem*/, "callNeo")    
}
```
The aim of this test is to check that newly-added field inside our interoperable API is accessible as expected.

Reply:
Done.